### DISCUSSION TITLE
Chevron | alignment x text size

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1392] [Proposals]
### Goal

Verificar a inconsistência e atualizar alinhando o icon-chevron ao texto do componente.

### Context

Notamos que o icon-chevron no componente Botão Link não está alinhado ao centro na atualização que subiu no aplicativo atual da Vivo.

### Impact

Manter a consistência visual.

### Discussion Questions

ND

### Do you have a visual proposal?

https://www.figma.com/file/NKjZPayMwravBsOEv7ZnFh/Alignment-x-Text-size-(chevron)?type=design&node-id=1444%3A9206&mode=design&t=bY7LjZP0paJyy8T1-1

**Comment by yceballost** [comment]
Reported as a bug to developer team, thanks Anne!
